extensions/interactions/ItemSelectionInput/ItemSelectionInput.html

```diff
@@ -1,13 +1,76 @@
+<style>
+  /* The following is needed so that the radio button does not show */
+  input[type="radio"] {
+    display: none;
+  }
+
+  md-checkbox {
```

Comment:
Also too broad -- this will apply throughout the entire site. Could you use a class prefixed with .item-selection-input instead?